applications/callflow/src/module/cf_temporal_route.hrl

```diff
@@ -51,6 +52,7 @@
               ,ordinal = ?RULE_DEFAULT_ORDINAL :: ordinal()
               ,month = ?RULE_DEFAULT_MONTH :: kz_time:month()
               ,start_date = ?RULE_DEFAULT_START_DATE :: kz_time:date()
+              ,end_date = ?RULE_DEFAULT_END_DATE :: kz_time:date()
```

Comment:
Should default to `'undefined'` instead of the tuple. Type would be `kz_time:date() | 'undefined'`